snack-shop-bff/src/orders/orders.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Controller, Get } from '@nestjs/common';
+
+@Controller('orders')
+export class OrdersController {
```

Comment:
Thanks for adding this endpoint as well! It made me realize we haven't explicitly talked about which approach to use as naming convention for the endpoints, be it plural or singular for resources. There's no actual right or wrong from a RESTful point of view, yet I like that you chose plural for resource collections - from what I've seen it's more frequently used, and I personally think it's more human readable this way.